cluster/src/main/java/io/scalecube/cluster/fdetector/FailureDetector.java

```diff
@@ -118,15 +115,15 @@ public void setMembers(Collection<Address> members) {
     List<Address> list = new ArrayList<>(set);
     Collections.shuffle(list);
     this.members = list;
-    LOGGER.debug("Set cluster members[{}]: {}", this.members.size(), this.members);
+    LOGGER.debug("Updated monitored members[{}]: {}", this.members.size(), this.members);
   }
 
   @Override
   public void start() {
     onPingRequestSubscriber = Subscribers.create(new OnPingRequestAction());
     transport.listen().observeOn(scheduler)
         .filter(PING_FILTER)
-        .filter(targetFilter(transport.address()))
+        .filter(targetFilter())
```

Comment:
I think this targetFilter is useless overall. Algorithm can't send Ping not to you. So basically just filtering out this case not valid.